packages/terra-layout/src/_LayoutSlidePanel.jsx

```diff
@@ -151,7 +151,7 @@ class LayoutSlidePanel extends React.Component {
         </div>
         <OverlayContainer className={cx('content')}>
           <Overlay isRelativeToContainer onRequestClose={onToggle} isOpen={isOverlayOpen} backgroundStyle={overlayBackground} />
-          <main className={cx('main-container')}>
+          <main id="terra-main" tabIndex="-1" className={cx('main-container')}>
```

Comment:
Should we be providing a data-attribute for this instead? Like `data-terra-layout-main`? In terra-table we pass [`data-terra-table-cell`](https://github.com/cerner/terra-core/blob/master/packages/terra-table/src/TableCell.jsx#L17)

Reply:
Got it changed. I like that a lot better than having the id on the main tag!